### DISCUSSION TITLE
How to get the result of how many records have been deleted?

### DISCUSSION THREAD

**Discussion by yf-hk** [mikro-orm/mikro-orm#2271] [Q&A] [answered]
For example, I would like to execute a query to delete a few records:
```js
this.repo.removeAndFlush(entities[]);
```
How can I know if the deletion was successful, and how many rows affacted?

Many thanks! 🙏

**Comment by B4nan** [comment, marked as answer]
This is not possible, you can use `nativeDelete()` if you need such number. `removeAndFlush` is just a shortcut for `remove()` followed by `flush()` - and the latter does not work just with one entity/table, it handles all of them (and all of create/update/delete operations).

> How can I know if the deletion was successful, and how many rows affacted?

Deletion is always successful, the question is only whether some of the entities were already deleted before (e.g. by some other request).
